### DISCUSSION TITLE
Unusual Trend in Validation Loss

### DISCUSSION THREAD

**Discussion by SidPad** [microsoft/MoCapAct#14] [Q&A]
Hi,

Thanks for your help in advance.
I tried to reproduce the result for the motion completion, I trained the GPT policy and visualized the action prediction in MuJoCo. I stopped the training early, thinking that I might have made a mistake since the validation losses did not follow the usual trend (following a decreasing trend) and there might have been a possibility of overfitting.
However, the visualization in MuJoCo looked fine, indicating that the model has learnt to predict the actions quite well.
Is there any reason why the validation losses are misleading? Or am I interpreting them wrong?
![MotionCompletionGraph](https://github.com/microsoft/MoCapAct/assets/56720486/585cb0cc-b086-42c8-80a0-7781f903ce87)

Please find the reproduced results attached to this message. The model was trained on the 600 GB dataset and I have used different values of learning rate, batch_size, and validation frequency, but the increasing trend is observed in most.

Thanks again for your help.
